client/src/components/VisualizerMenu/VizMenuItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { ForwardedRef, forwardRef, useEffect, useState } from 'react';
+import { MenuItem } from 'src/styles/VizMenuItemStyle';
+import { VizMenuProps } from 'src/types';
+
+function VizMenuItem(
+  { id, songName, createClickHandler, index }: VizMenuProps,
+  ref: ForwardedRef<HTMLDivElement>,
+) {
+  const clickHandler = createClickHandler;
+  const [curBkgdColor, setBkgdColor] = useState<string>('#1ab26b');
+
+  useEffect(() => {
+    if (index) {
+      switch (index % 5) {
+        case 1:
+          setBkgdColor('#1c4347');
```

Comment:
The hex background colors are acting somewhat like magic numbers here (https://codetuts.dev/magic-numbers-in-programming/). I know you're working this project solo so this is basically a non-issue... but I'd still recommend making these values into brief, descriptive variables instead.